Microsoft.WindowsAzure.Storage/includes/wascore/timer_handler.h

```diff
@@ -0,0 +1,82 @@
+// -----------------------------------------------------------------------------------------
+// <copyright file="timer_handler.h" company="Microsoft">
+//    Copyright 2018 Microsoft Corporation
```

Comment:
2019?

Reply:
Per offline discussion, this is the time that this file is created, not the time added to Github.